### PR TITLE
Team page

### PR DESCRIPTION
**Pull request by Sambhav-Sam**
ecell

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/material": "^5.11.8",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.7.1",
```
Comment (sanskaromar) [inline client/package.json:15]: You icons from here for consistency in code as your other Team Mates are preferring this as well.

File: client/package.json [hunk 2]
```diff
@@ -3,11 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/material": "^5.11.8",
```
Comment (sanskaromar) [inline client/package.json:9]: We don't need MUI in our project as we are already using tailwind, and I don't thing MUI is required here.

File: client/public/index.html [hunk 3]
```diff
@@ -9,21 +9,143 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <script src="https://cdn.tailwindcss.com"></script>
```
Comment (sanskaromar) [inline client/public/index.html:12]: We don't need tailwind cdn. Tailwind is installed in our project.

File: client/public/index.html [hunk 4]
```diff
@@ -9,21 +9,143 @@
...
+        .info {
+          width: 300px;
+          border-radius: 2px;
+          height: 38px;
+          background-color: #cad2c5;
+          display: flex;
+          align-items: center;
+        }
+      }
+    </style>
```
Comment (sanskaromar) [inline client/public/index.html:146]: index.html is not the best place to add CSS.
Try to use tailwind classes whenever possible.

File: client/public/index.html [hunk 5]
```diff
@@ -9,21 +9,143 @@
...
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>React App</title>
```
Comment (sanskaromar) [inline client/public/index.html:149]: You edited everything but one thing that was actually necessary. The title.

File: client/src/components/TeamPage/CardTest.js [hunk 6]
```diff
@@ -0,0 +1,204 @@
+import React from "react";
+import FacebookIcon from "@mui/icons-material/Facebook";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import TwitterIcon from "@mui/icons-material/Twitter";
```
Comment (sanskaromar) [inline client/src/components/TeamPage/CardTest.js:4]: Use react-icons

**Review by sanskaromar — CHANGES_REQUESTED** [review]

**Review by NeerajChatterjee — CHANGES_REQUESTED** [review]
As mentioned by Sanskar, remove the styling from index.html and use react-icons. Update the needful.

**Review by vishwa511 — CHANGES_REQUESTED** [review]
Make the icons more attractive on hover,  you can use scale property, since exact design is not provided use your own creativity
also the circle of same size covering the icon is not looking good, you can make it a bit bigger and can transform on hover
snape

**Review by NeerajChatterjee — DISMISSED** [review]

File: client/public/index.html [hunk 7]
```diff
@@ -10,25 +10,20 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <title>Design Team</title>
```
Comment (sanskaromar) [inline client/public/index.html:26]: This page is not for the design team. It's for Renaissance 2k23-Team.

File: client/src/components/TeamPage/CardTest.css [hunk 8]
```diff
@@ -0,0 +1,136 @@
+body {
```
Comment (sanskaromar) [inline client/src/components/TeamPage/CardTest.css:1]: Most of the things in this file could have been easily done with Tailwind.

File: client/src/components/TeamPage/CardTest.js [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+              </a>
+              <a href="">
+                <AiFillLinkedin className="space" />
+              </a>
+              <a href="">
+                <AiFillYoutube className="space" />
+              </a>
+            </div>
+          </div>
+        </div>
```
Comment (sanskaromar) [inline client/src/components/TeamPage/CardTest.js:47]: Have you heard about components? Wouldn't it be easier to make this a separate component, and we pass their details via props whenever we want to add a new person?

**Review by sanskaromar — CHANGES_REQUESTED** [review]

**Review by sanskaromar — APPROVED** [review]

**Pull request merged**